cli/internal/cmd/configfetchmeasurements_test.go

```diff
@@ -154,115 +151,13 @@ func newTestClient(fn roundTripFunc) *http.Client {
 }
 
 func TestConfigFetchMeasurements(t *testing.T) {
```

Comment:
Can you add test cases to check:
* Errors other than Rekor errors are not skipped
* No error actually passes correctly